src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator {
```
тоді всі наслідники будуть імплементувати інтерфейс за замовчуванням